plenum/test/checkpoints/test_message_outside_watermark1.py

```diff
@@ -1,16 +1,34 @@
 import math
 
+import pytest
+
+from stp_core.common.log import getlogger
 from stp_core.loop.eventually import eventually
 
 from plenum.test import waits
-from plenum.test.delayers import ppDelay
+from plenum.test.delayers import ppDelay, pDelay
 from plenum.test.helper import sendReqsToNodesAndVerifySuffReplies
 from plenum.test.test_node import getNonPrimaryReplicas, getPrimaryReplica
 
 
 TestRunningTimeLimitSec = 300
 
 
+logger = getlogger()
+
+
+@pytest.fixture(scope="module")
+def tconf(tconf, request):
```

Comment:
Because this test will send many request with delays, it can intermittently lead to failures due to requests not completing in time because of view change